packages/web/src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Carousel from 'react-bootstrap/Carousel';
+import './styles.scss';
+
+type Props = {
+  images: string[] | undefined;
```

Comment:
I don't think this makes sense really. I don't think someone would explicitly pass undefined as a prop value, and I don't see it can happen from the usage. It's a bit different from a prop being explicitly set to null from the outside. If you think prop might not be passed you can make it optional and set a default value for it instead, e.g.

type Props = {
  images?: string[];
}

export const Gallery = ({ images = [] }: Props) => {
  ...